orb-ui/src/engine/mod.rs

```diff
@@ -70,9 +70,6 @@ macro_rules! event_enum {
                 $(#[doc = $doc])?
                 fn $method(&self, $($($field: $ty,)*)?);
             )*
-
-            /// Returns a new handler to the shared queue.
```

Comment:
It seems to me that it is convenient for users to be able to clone the engine. You have presented a synchronized "proxy" object to acces the sound, why not allow people to clone it so that they can pass it to various tasks/threads?

Reply:
I just thought I would remove unused code